code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -227,7 +227,7 @@
 /obj/machinery/smartfridge/proc/visible_items()
 	var/component_part_count = 0
 	for(var/datum/stock_part/datum_part in component_parts)
```

Comment:
This code looks useless anyways, it looks like a hack back when stock parts where objects inside the contents but they're not anymore. 

I just checked on live and an empty smart fridge has one content, the circuitboard

Reply:
In fact that's the only reason why the original code worked, it was doing `contents - [contents it doesn't wanna show]`, now it's just doing `contents.len - 1`